app/src/main/java/org/oppia/app/player/exploration/ExplorationFragmentPresenter.kt

```diff
@@ -33,7 +50,24 @@ class ExplorationFragmentPresenter @Inject constructor(
     }
     return binding
   }
+  private fun getProfileData(): LiveData<StoryTextSize> {
```

Comment:
Ditto here: newline above to separate the methods.

Reply:
Done.